docs/content/latest/yugabyte-cloud/cloud-monitor/cluster-tables.md

```diff
@@ -1,4 +1,4 @@
----
+<!---
```

Comment:
You're intentionally omitting this file from the hierarchy at this point, I take it?